Cargo.toml

```diff
@@ -136,7 +141,12 @@ required-features = ["macros"]
 
 [[example]]
 name = "db_remember"
-required-features = ["sqlite-storage", "redis-storage", "bincode-serializer", "macros"]
+required-features = [
+  "sqlite-storage",
+  "redis-storage",
+  "bincode-serializer",
+  "macros",
+]
```

Comment:
Does your IDE make such kind of formatting? As far as I can tell, `cargo fmt` doesn't format `.toml` files. Since we might have different local formatters, I suggest leaving the file as-is.

Reply:
Ok, I will revert the changes. I use vscode with [Even Better TOML](https://marketplace.visualstudio.com/items?itemName=tamasfe.even-better-toml) extension. There is [taplo](https://taplo.tamasfe.dev/configuration/formatter-options.html) tool inside, so I suggest defining a config file in a separate PR. It will allow all developers not to spend time on this.

Reply:
Done